elements/pfe-progress-steps/src/pfe-progress-steps.scss

```diff
@@ -0,0 +1,81 @@
+@import "_shared-assets";
+
+$LOCAL: progress-steps;
+
+$LOCAL-VARIABLES: map-deep-merge($LOCAL-VARIABLES, (
+  progress-bar: (
+    color: pfe-var(ui--border--lighter),
+    color--active: pfe-var(feedback--info),
+    thickness: pfe-var(ui--border-width--md)
+  ),
+));
+
+:host {
+  display: block;
+  display: flex;
+  width: auto;
+  justify-content: space-between;
+  position: relative;
+  align-items: flex-start;
+  
+  // @TODO This definitely would need some IE love since it's driven by CSS variables so I'm just not displaying it in IE for now.
+  @include browser-query(ie11) {
+    display: none;
+  }
+}
+
+// Make everything vertical
+:host([vertical]) {
+  flex-direction: column;
+}
+
+:host([hidden]) {
+  display: none;
+}
+
+.pfe-progress-steps__progress-bar {
+  display: flex;
+  position: absolute;
+
+  top: calc(
+    var(--pfe-progress-steps-item__circle--size--active, #{pfe-fetch(size--active, $region: circle)}) -
+    var(--pfe-progress-steps-item__circle--size, #{pfe-fetch(size, $region: circle)}) +
+    #{pfe-local(thickness, $region: progress-bar)}
+  ); // calc(32px - 20px + 2px)
+
+  left: calc(#{pfe-local(size--first, $region: item)} / 2);
+
+  height: pfe-local(thickness, $region: progress-bar); // 2px
+  width: calc(100% - #{pfe-local(size--first, $region: item)} / 2 - #{pfe-local(size--last, $region: item)} / 2);
+  background-color: pfe-local(color, $region: progress-bar);
+
+  :host([vertical]) & {
+    width: pfe-local(thickness, $region: progress-bar);
+
+    // Position the progress bar correctly over the circle
+    top: calc(
+      var(
+        --pfe-progress-steps-item__circle--size--active,
```

Comment:
```suggestion
        #{pfe-local(size--active, $region: circle)},
```

Reply:
Notice this is referencing progress-steps-item vs. progress-steps which is what the local variable is set to. That is why this is hardcoded rather than using the function.